src/services/TextureLoader.js

```diff
@@ -72,6 +72,10 @@ export class TextureLoader extends AbstractService {
       let progress = 0;
       onProgress && onProgress(progress);
 
+      if (typeof this.config.addRequestHeaders === 'function') {
+        this.loader = this.config.addRequestHeaders(this.loader);
```

Comment:
this should be done once in the constructor

Reply:
When I move it to the constructor it won't work with Authorization header with a valid bearer token. The TextureLoader class is created within the constructor of the viewer. So the callback function will never be called before each loader.load and the bearer token will be invalid.

Reply:
@hcoppen can you give an example of your usage ?

If it must be a function I would prefer this : 

```js
if (this.config.getRequestHeaders) {
  this.loader.setRequestHeaders(this.config.getRequestHeaders());
}
```

Reply:
we can also choose to return only the header object. 

`  var viewer = new PhotoSphereViewer.Viewer({
    container : 'photosphere',
    addRequestHeaders: (loader) => {
        loader.setRequestHeader( { 
          Authorization: "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI0YjIxNmJlYi02M......"
          } );
        return loader;
      },
    adapter   : [PhotoSphereViewer.EquirectangularTilesAdapter, {
      showErrorTile: true,
    }],
    loadingImg: 'assets/photosphere-logo.gif',
    navbar    : [
      'autorotate', 'zoom',
      ],
  });`